javascript/javascript2/week1/js2week1.js

```diff
@@ -0,0 +1,66 @@
+// Problem solving cardio
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+
+function findShorty() {
+    const shortestWord = danishWords[0].length
+    for (let i = 0; i < danishWords.length; i++) {
+        const word = danishWords[i].length
+        if (word < shortestWord) {
+            word = shortestWord
+        }
+        return shortestWord
+    }
+}
+
+console.log (findShorty(danishWords));
+
+// Find and count the Danish letters
+
+
+const danishString = "Jeg har en blå bil";
+const danishString2 = "Blå grød med røde bær";
+
+
+function findDanishLetter(str, letter,) 
+{
+ var letterCount = 0;
+ for (let i = 0; i < str.length; i++) 
+ {
+    if (str.charAt(i) == letter) 
+      {
+      letterCount += 1;
+      }
+  }
+  return letterCount;
+}
+
+console.log(findDanishLetter(danishString2, "å"))
+
+// Spirit animal name generator
+
+const animalArray = ["The Crazy Wolf", "The Mad Cow's cousin", "The Smoking Lion", "The Pharo Cat", "The Teaching Raccon", "The Wise Horse", "The Shamfull Dog", "The Mousy Turtle"]
+
+
+
+function getInputValue(){
```

Comment:
this function is calling when you click on btn button, so you shouldn't add another event listener inside this function for listening to the click event.